packages/notifications/push/amplify_push_notifications/pubspec.yaml

```
@@ -12,6 +12,7 @@ dependencies:
   amplify_auth_cognito: ">=1.0.0-next.4 <1.0.0-next.5"
   amplify_core: ">=1.0.0-next.0 <1.0.0-next.1"
   amplify_secure_storage: ^0.1.0
+  async: ^2.11.0
```

Should be in sync of [this](https://github.com/aws-amplify/amplify-flutter/blob/next/aft.yaml#L9), can run `aft constraints` to align.

```suggestion
  async: ^2.10.0
```

---

Nice, updated other versions too based on `aft constraints`